_data/food.yml

```diff
@@ -39,8 +39,10 @@ websites:
   - name: Starbucks
     url: https://www.starbucks.com/
     img: starbucks.png
-    twitter: Starbucks
-    facebook: Starbucks
+    tfa:
+      - sms
+      - phone
```

Comment:
Are you sure they do phone calls?

Reply:
It appeared to be an option but I just checked and it's only an option during setup, all future codes are sent over sms.

Reply:
```suggestion
```